inductiva/resources/machine_cluster.py

```diff
@@ -32,7 +32,7 @@ def __init__(self,
               Check https://cloud.google.com/compute/docs/machine-resource for
               information about machine types.
             num_machines: The number of virtual machines to launch.
-            disk_size_gb: The size of the disk in GB, recommended min. is 60 GB.
+            disk_size_gb: The size of the disk in GB.
```

Comment:
wdyt?
If you like it, make it consistent across everything.

```suggestion
            data_size_gb: The size of the disk for user data (in GB).
```